### PR TITLE
fix #79064 nurgsm.com

### LINKED ISSUE #79064: nurgsm.com
### Issue URL (Anti Adblock Script)
 [https://nurgsm.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnurgsm.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.23
Browser: | Opera
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>Peter Lowe's Blocklist<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#79064

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -98,6 +98,8 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+nurgsm.com##div[id^="sgpb-popup-dialog"]
+nurgsm.com##div[class$="sgpb-popup-overlay"]
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:102]:
```
nurgsm.com##.sgpb-popup-overlay
nurgsm.com##.sgpb-popup-dialog-main-div-wrapper
```
Why not? Also duplicate these rules to the Social - `popups.txt`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/popups.txt:102]: Aren't the rules too big then?

**Review by zloyden — CHANGES_REQUESTED** [review]
I can see a leftover. This `nurgsm.com##.ai_widget` should be ok.

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt]: For now, it's ok.

**Review by zloyden — APPROVED** [review]

**Pull request merged**